dist/index.html

```diff
@@ -38,9 +38,23 @@ <h1>
         </div>
     </header>
     <footer class="footer-section" id="partners">
-        
-
-    </footer>>
+        <h3>Partners</h3>
+        <div class="partner-names">
+            <div class="partner">
+                <img src="./Assets/kalutu.jpeg" alt="Partner 1">
```

Comment:
Consider adding partners details in the alt place like  <img src="./Assets/kalutu.jpeg" alt="A picture of Mr Daniel Bobai as one of the partners"> so that when the picture fails to load, a user can see what was broken. It will also help those using screen reader.